conda_build/metadata.py

```diff
@@ -1442,7 +1442,7 @@ def name(self) -> str:
 
     def version(self) -> str:
         version = self.get_value("package/version", "")
-        if not version and self.final:
+        if not version and not self.get_section("outputs") and self.final:
             sys.exit("Error: package/version missing in: %r" % self.meta_path)
```

Comment:
I guess this was there before, and not "raise some CondaBuildError" 🦺

Reply:
Correct